python/__init__.py.in

```diff
@@ -21,8 +25,21 @@ if sys.version_info >= (3, 8) and sys.platform == "win32":
     for rel_path in F3D_RELATIVE_DLLS:
         os.add_dll_directory((root / rel_path).resolve())
 
+# Store existing symbols before the import
+_existing_symbols = set(globals().keys())
+
 from .pyf3d import *
 
+# Automatically populate __all__ with symbols imported from pyf3d
+__all__ = [
+    name
+    for name in globals().keys()
+    if not name.startswith("_") and name not in _existing_symbols
+]
```

Comment:
Should we do that to avoid having to use `_existing_symbols` at all?
```suggestion
__all__ = [name for name in dir(pyf3d) if not name.startswith("_")]
```
the only difference I can see is that `__all__` then does not contain `'pyf3d'`, not sure if that matters